src/app/Data/Migrations/PetHomeContextModelSnapshot.cs

```diff
@@ -0,0 +1,230 @@
+﻿// <auto-generated />
+using System;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Infrastructure;
+using Microsoft.EntityFrameworkCore.Metadata;
+using Microsoft.EntityFrameworkCore.Storage.ValueConversion;
+using app.Data;
+
+namespace app.Data.Migrations
+{
+    [DbContext(typeof(PetHomeContext))]
+    partial class PetHomeContextModelSnapshot : ModelSnapshot
+    {
+        protected override void BuildModel(ModelBuilder modelBuilder)
+        {
+#pragma warning disable 612, 618
```

Comment:
i assume this is generated code? if it isn't, then #pragmas should have comments with justifications.